runtime/ui/src/sys/linux/cuivectorfont.cpp

```diff
@@ -29,66 +31,556 @@ define_holder(ILTTexInterface, pTexInterface);
 static ILTClient *ilt_client;
 define_holder(ILTClient, ilt_client);
 
+#define WIDTHBYTES( nBitCount, nPixelWidth ) (((( uint32 )nBitCount * nPixelWidth + 7) / 8 + 3) & ~3 )
 
 class InstalledFontFace
 {
+public:
+
+	InstalledFontFace()
+	{
+		m_nHeight = 0;
+		m_ttf_buffer = nullptr;
+	}
+
+	~InstalledFontFace()
+	{
+		Term();
+	}
+
+	// Intialize the font face, from optional font file.
+	bool Init(char const* pszFontFile, char const* pszFontFace, int nHeight, LTFontParams* fontParams);
+	void Term();
+
+	char const* GetFontFace() { return m_sFontFace.c_str(); }
+	int				GetHeight() { return m_nHeight; }
+	unsigned char* GetBuffer() { return m_ttf_buffer; }
+
+private:
+
+	std::string		m_sFontFace;
+	int				m_nHeight;
+	unsigned char* m_ttf_buffer;
 };
 
-bool CUIVectorFont::CreateFontTextureAndTable(InstalledFontFace&,char const*,bool bMakeMap)
+bool InstalledFontFace::Init(char const* pszFontFile, char const* pszFontFace, int nHeight, LTFontParams* fontParams)
+{
+	FILE* fp;
```

Comment:
fp is unused... and I think it duplicates the purpose of pStream
which seams to be used...